lib/configure.js

```diff
@@ -48,6 +48,7 @@ function configure (gyp, argv, callback) {
         }
       } else {
         log.verbose('`which` succeeded', python, execPath)
+        python = execPath
```

Comment:
What does this fix?  If `which('python')` succeeds, then by definition python is on the PATH, right?